python/setup.py

```diff
@@ -35,59 +42,143 @@
     cuda_include_dir = os.path.join(os.environ.get('CUDA_HOME'), 'include')
 
 
-rmm_include_dir = '/include'
-rmm_lib_dir = '/lib'
-
-# including cumlprims dirs, in case rmm paths change in the future
-cumlprims_include_dir = '/include'
-cumlprims_lib_dir = '/lib'
+conda_lib_dir = os.path.normpath(sys.prefix) + '/lib'
+conda_include_dir = os.path.normpath(sys.prefix) + '/include'
 
 if os.environ.get('CONDA_PREFIX', None):
     conda_prefix = os.environ.get('CONDA_PREFIX')
-    rmm_include_dir = conda_prefix + rmm_include_dir
-    rmm_lib_dir = conda_prefix + rmm_lib_dir
+    conda_include_dir = conda_prefix + '/include'
+    conda_lib_dir = conda_prefix + '/lib'
```

Comment:
You shouldn't need to do anything CONDA specific, if you just use `sys.prefix` it has the CONDA environment or normal venv.

Reply:
Another of my PRs was updating this, can’t remember which.... will apply the fix here as well